Sources/TuistCore/ContentHashing/ContentHasher.swift

```diff
@@ -50,19 +50,26 @@ public final class ContentHasher: ContentHashing {
 
     public func hash(path filePath: AbsolutePath) throws -> String {
         if fileHandler.isFolder(filePath) {
-            let paths = try fileHandler.contentsOfDirectory(filePath)
-            let sortedPaths = paths.sorted(by: { $0 < $1 })
-            return try sortedPaths.map { try hash(path: $0) }.joined(separator: "-")
+            return try fileHandler
+                .contentsOfDirectory(filePath)
+                .filter { $0.basename.uppercased() != ".DS_STORE" }
```

Comment:
I think this filtering is worth extracting into a component that we can later extend with more patterns.